docs/en/revocation-lists.rst

```diff
@@ -100,6 +100,9 @@ Credential Revocation Flows can start under different scenarios, such as:
     - The Authentic Sources that for any administrative reasons update one or more attributes of a User, shall inform the Issuer of related Credentials. 
     - The Issuers, for technical security reasons (e.g. in the case of compromised cryptographic keys, death of the User, etc.), can decide to revoke the Credentials.
 
+The Credential in some cases MUST be considered valid (thus is still accurate, useful and valid) for the period before the revocation: a driving license, revoked due expiration or other reasons, is still valid for the period before the revocation.
```

Comment:
```suggestion
There are cases where a Credential MAY be considered valid (thus is still accurate, useful and valid) for a period after the revocation: a driving license, revoked due expiration or other reasons, MAY be still valid for the period after the revocation.
```

is this what you want to say?
could you please bring also legal evidences as example according to the italian regulation?

Reply:
Not really, considering the example of Residence Certificate: I may be resident in Rome up to 31st of December 2023. I change my residency to Milan from 1st of January 2024, thus the previous residency certificate gets revoked.
The residency certificate revoked (Rome's residency) is still valid for any activity/action  related to year 2023 and before (e.g. I need to pay taxes in Rome city for 2023, and these taxes are paid in 2024).

Reply:
As a general comment, the verifier policy always determines "validity", issuer's provide guidance and integrity protection, the "validity" is always in the eye of the verifier.

You might consider language like:

A verifier MAY accept credentials presented outside of their validity period, or with revoked or suspended status.
Verifier policies are out of scope for this document.